types/defines/queues.d.ts

```diff
@@ -44,4 +54,11 @@ interface Queue<Body = any> {
    * @returns A promise that resolves when the message is confirmed to be written to disk.
    */
   send(message: Body): Promise<void>;
+
+  /**
+   * Sends a batch of messages to the Queue.
+   * @param messages Each item in the array must be supported by the [structured clone algorithm](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm#supported_types). A batch can contain up to 100 messages, though items are limited to 128 KB each, and the total size of the array cannot exceed 256 KB.
+   * @returns A promise that resolves when the messages are confirmed to be written to disk.
+   */
+  sendBatch(messages: Iterable<MessageSendRequest<Body>>[]): Promise<void>;
```

Comment:
But then again I'm also not sure that it's idiomatic for us to define this as an `Iterable`... Perhaps it's better to just keep it as an `MessageSendRequest<Body>[]`. I don't see any other types in workers-types that are defined as an `Iterable`, and the closest analogs I looked at (`HeadersInit` and `URLSearchParamsInit`) just appear to just be defined as arrays.

Sorry for leading you astray on that part!

Reply:
cc @mrbbot to confirm -- is a `jsg::Sequence<T>` best defined in typescript types as a `T[]`?

Reply:
Thanks @mrbbot !